src/WebJobs.Script/Config/ApplicationHostOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+/// <summary>
+/// This is a copy of the <see cref="ScriptApplicationHostOptions"/> excluding the <see cref="IServiceProvider"/> property.
+/// The purpose of this class is to provide a subset of the <see cref="ScriptApplicationHostOptions"/> that can be serialized
+/// when used in API responses.
+/// </summary>
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public class ApplicationHostOptions
```

Comment:
Open to better naming suggestions here: @fabiocav @kshyju

Reply:
It would be helpful to avoid the `Options` suffix here to avoid confusing this with the other types used with the Options pattern.

Even something very specific like `HostRestartResponse`, `ApplicationHostModel`, etc. Anything without the `Options` suffix works

Reply:
I'll go with `ApplicationHostModel` - I don't want to use HostRestartResponse as this could apply to any API that wants to respond with this information

Reply:
> I don't want to use HostRestartResponse as this could apply to any API that wants to respond with this information.

Do we have use cases? I'm good with the name you've picked, but you can also make the argument that this response could include more than the information contained in `ApplicationHostOptions`.

Approving as this is not a blocker.

Reply:
That's a fair point; no use cases I am aware of - just future proofing. The CI needs to run again so I can make the change any who :)